baseline/OSPS-SA.yaml

```diff
@@ -68,6 +68,12 @@ controls:
       - reference-id: UKSSCOP
         identifiers:
           - 1.4
+      - reference-id: 800-161
+        identifiers:
+          - CM-2
+          - PL-8
+          - RA-3
+          - SA-15          
```

Comment:
As I read it, this OSPS-SA-01 is about the actors/actions within the released software, whereas NIST's SA-15 is more about the process for building it.

Reply:
- SA-15 reads "Providing documented and formalized development processes to guide internal and system integrator developers is critical to the enterprise’s efforts to effectively mitigate cybersecurity risks throughout the supply chain."

These design docs/refarchs are the documents a consumer would use to perform this task and justify their choices in their risk management activities.